streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -171,85 +158,67 @@ public void before() {
                 "threadId-StreamThread-0",
                 mockClientSupplier,
                 null,
-                null,
                 logContext,
                 mockTime
             );
         nonEosMockProducer = mockClientSupplier.producers.get(0);
 
-        eosAlphaMockClientSupplier.setCluster(cluster);
-        eosAlphaMockClientSupplier.setApplicationIdForProducer("appId");
-        eosAlphaStreamsProducer =
+        eosMockClientSupplier.setCluster(cluster);
```

Comment:
Many EOS tests where not EOSv1 vs EOSv2 specific, but used `eosAlphaXxx` variables -- updating these to use `eosBetaXxx` (renamed to just `eosXxxx`) -- more of this below.